guardian/floodgate
# Bump cross spawn

## What does this change?

Fixes https://github.com/advisories/GHSA-3xgq-45jj-v275.

## How to test

How to test

- Build commands should work as before
- Deploy to CODE. Frontend should work as expected
